statsmodels/stats/inter_rater.py

```diff
@@ -109,7 +109,8 @@ def aggregate_raters(data, n_cat=None):
     arr : nd_array, (n_rows, n_cat)
         Contains counts of raters that assigned a category level to individuals.
         Subjects are in rows, category levels in columns.
-
+    categories : nd_array, (category_levels)
```

Comment:
(category_levels) -> (n_category_levels,)

to clear indicate this is a size tuple.

Reply:
Ahh sorry about that. Fixed!